cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -0,0 +1,100 @@
+# HTTP Cache configuration.
+#
+# See cdo-varnish/README.md for more information on the configuration format.
+class HttpCache
+  STATIC_ASSETS = {
+    # For static-asset extensions, don't forward any cookies or headers.
+    path: %w(cur pdf png gif jpeg jpg ico mp3 swf css js).map{|ext| "*.#{ext}"},
+    headers: [],
+    cookies: 'none'
+  }
+
+# Host header is needed for Varnish to delegate to pegasus/dashboard depending on the host.
+# Accept-Language header is needed to cache language-specific pages.
+  ALL_HEADERS = %w(Host Accept-Language)
```

Comment:
Could you please document what this means?  ALL_HEADERS is again confusing here since it isn't all headers but only a few headers.

Are these really the only headers passed on to the Rails frontend, or am I misinterpreting what they mean?  It seems like ALL_HEADERS is used for a lot of paths below.  There are lots of important request headers that are being stripped here that I would be surprised not to find in Rails controllers (HOST, Cookie, If-Modified-Since, etc) which make me think I must be misinterpreting things.  

(This is another place where philosophically I wish we could list headers that we've analyzed and know are safe to strip, versus listed a small set which we think we need.)